yb-voyager/cmd/exportSchema.go

```diff
@@ -126,6 +126,8 @@ func init() {
 
 	BoolVar(exportSchemaCmd.Flags(), &source.CommentsOnObjects, "comments-on-objects", false,
 		"enable export of comments associated with database objects (default false)")
+	exportSchemaCmd.Flags().StringVar(&source.ExportObjects, "object-list", "",
```

Comment:
Let's name this argument as `--object-type-list`. Fix all your code changes to distinguish between an `object` vs an `object-type`.

`source.ExportObjectTypes`.